src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryLoggerExtensions.cs

```diff
@@ -47,6 +48,10 @@ static AntiforgeryLoggerExtensions()
                 LogLevel.Error,
                 7,
                 "An exception was thrown while deserializing the token.");
+            _responseNotCached = LoggerMessage.Define(
+                LogLevel.Debug,
+                8,
+                "The 'Cache-Control' and 'Pragma' headers of the response have been set as 'no-cache'.");
```

Comment:
Rather than doing this, I think it would be much better to check if the `Cache-Control` and `Pragma` headers were already set and log a **warning** log message if they are.